src/pages/components/BrowserContextMenu.qml

```diff
@@ -28,6 +29,11 @@ Rectangle {
     readonly property bool active: visible
     readonly property bool landscape: width > height
 
+    readonly property string url: !isJavascriptFunction ? linkHref : ""
+    readonly property bool isJavascriptFunction: linkProtocol === "javascript"
+    readonly property bool isLink: !isJavascriptFunction && linkHref.length > 0 && imageSrc.length === 0
```

Comment:
@jpetrell 
Having either "Open link" or "Save link" would be definitely nice. However, that would require much more changes and some new apis. Something to be fixed in future.

Long pressing on top of a photo you'll get "Open image in new tab" and "Save to gallery". Identifying what's behind a link requires some sort of partial prefetching of a href content. Engines have been handling prefetching while engine is idling for some time (link element) but it really cannot/should not be arbitrary prefetching. 

I'm fine with dropping "link" suffix. 

Google Chrome doesn't have "Open link" options in context menu but they save "Open in new tab" and "Save link". Both Android Firefox and Google Chrome have more context information about the data. We added "Open link" to the menu as somebody was asking about it => personally I could even think of dropping it. One use case that I can think of is that rather clicking a suspicious link directly, you'd like to check that link is somewhat sane.

Furthermore, Google Chrome logic hides "Save link" when long pressing a article on  m.yle.fi/uutiset.  However, they still have "Copy link address" which doesn't make any sense as that is a javascript function. Also "Open in new tab" works => I just hide it as it is trying to open "void(0)" and it just doesn't work in our context => improvements needed on the embedlite-components side.

What comes to context menu handling in general there's plenty of room to improve when comparing to competitors browsers. The Metro code (Windows Firefox) didn't really help here: https://github.com/tmeshkova/embedlite-components/pull/94 => they have some what similar approach as we.

As a summary about this quick fix, this handles basic stuff :)

@siteshwar :
tel, mailto, geo etc are valid links in this context. Thus, we should show them normally. However, handling of those could be improved a lot. I'll amend some basic cases.